### PR TITLE
Feature/tf 733 brs linux64 build

### PR DESCRIPTION
**Pull request by GabrielHomewoodUL**
## Summary

Update to build for more platforms

## Contributor Tasks

_These tasks are for the pull request creator to tick off._

- [ ] Relevant changelogs have been updated with user-visible changes
    - [ ] [TouchFree Windows](/ultraleap/touchfree/blob/-/CHANGELOG-windows.md)
    - [ ] [TouchFree BrightSign](/ultraleap/touchfree/blob/-/CHANGELOG-brightsign.md)
    - [ ] [TouchFree Web Tooling](/ultraleap/touchfree/blob/-/TF_Tooling_Web/CHANGELOG.md)

## Reviewer Tasks

_Add any instructions or tasks for the reviewer such as specific test considerations before this can be merged._

- [ ] Developer testing
- [x] Code reviewed
- [ ] Non-code assets reviewed
- [ ] Documentation reviewed - includes checking documentation requirements are met and not missing e.g., public API is commented

### CODE REVIEW

File: TF_Service_dotNet/TouchFree/TouchFreeLib.csproj [hunk 1]
```diff
@@ -5,19 +5,26 @@
...
 		</ContentWithTargetPath>
 	</ItemGroup>
 
 	<ItemGroup>
-		<ContentWithTargetPath Condition="'$(PlatformTarget)'=='ARM64'" Include="Plugins\LeapC\ARM64\*">
+		<ContentWithTargetPath Condition=" '$(BRIGHTSIGN)' == 'true' and '$(PlatformTarget)'=='ARM64' " Include="Plugins\LeapC\ARM64\*">
+			<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+			<TargetPath>%(Filename)%(Extension)</TargetPath>
+		</ContentWithTargetPath>
+	</ItemGroup>
```
Comment (DavidFee-UL) [inline TF_Service_dotNet/TouchFree/TouchFreeLib.csproj:24]: Is this a duplicate entry?

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -153,79 +153,66 @@ TF_Service::Wrapper::build:
...
     - if ($CI_COMMIT_REF_NAME -Match 'release/.*') {
-        $TOUCHFREE_SUFFIX = ''
+        $TOUCHFREE_SUFFIX = $DEFAULT_RELEASE_SUFFIX
       } else {
         $TOUCHFREE_SUFFIX = 'dev+' + ${env:CI_COMMIT_SHORT_SHA}
       }
 
-    - dotnet publish -p:VersionPrefix=$TOUCHFREE_VERSION -p:VersionSuffix=$TOUCHFREE_SUFFIX -c Release -r win-x64 "$CI_PROJECT_DIR/TF_Service_dotNet/TouchFree/TouchFreeLib.csproj"
-    - dotnet publish -p:VersionPrefix=$TOUCHFREE_VERSION -p:VersionSuffix=$TOUCHFREE_SUFFIX -c Release -r win-x64 "$CI_PROJECT_DIR/TF_Service_dotNet/TouchFree_Service/TouchFree_Service.csproj"
+    - dotnet publish -p:BRIGHTSIGN=$BRIGHTSIGN -p:VersionPrefix=$TOUCHFREE_VERSION -p:VersionSuffix=$TOUCHFREE_SUFFIX -c Release -r $RUNTIME "$CI_PROJECT_DIR/TF_Service_dotNet/TouchFree/TouchFreeLib.csproj"
```
Comment (DavidFee-UL) [inline .gitlab-ci.yml:174]: I can't find what the -p option does but I assume it's a variable definition. Have you any docs on this?

File: TF_Service_dotNet/TouchFree/TouchFreeLib.csproj (hunk 1 above)
Comment (DavidFee-UL) [reply TF_Service_dotNet/TouchFree/TouchFreeLib.csproj:24]: Ah I think I see: '$(PlatformTarget)'=='ARM64' and '$(PlatformTarget)'!='ARM64' seem redundant. So merge into a single group.

**Review by DavidFee-UL — CHANGES_REQUESTED** [review]

File: TF_Service_dotNet/TouchFree/TouchFreeLib.csproj (hunk 1 above)
Comment (GabrielHomewoodUL) [reply TF_Service_dotNet/TouchFree/TouchFreeLib.csproj:24]: I've added this as a placeholder for the x64 libs, so we can just add them and update the folder that the `$(PlatformTarget)!=ARM64` group points at

Comment (DavidFee-UL) [reply TF_Service_dotNet/TouchFree/TouchFreeLib.csproj:24]: Got you.

**Review by DavidFee-UL — APPROVED** [review]

**Pull request merged**